src/plugins/data_table/column_types/base.js

```diff
@@ -4,7 +4,7 @@ app.view.dataTable.columnType(function(columnType) {
   });
 
   columnType.nodeMatcher(function(config) {
-    return "." + Backdraft.Utils.toCSSClass(config.title);
+    return "." + Backdraft.Utils.toCSSClass(config.attr || config.title);
```

Comment:
and this file can just stay as config.title because notice it's only used when `!config.attr` (per line 3)